OpenRA.Game/Game.cs

```diff
@@ -368,7 +368,9 @@ public static void InitializeMod(string mod, Arguments args)
 			Sound.StopVideo();
 
 			ModData = new ModData(Mods[mod], Mods, true);
-			ExternalMods.Register(ModData.Manifest);
+			var manifest = ModData.Manifest;
+			if (!ExternalMods.ContainsKey(ExternalMod.MakeKey(manifest)))
```

Comment:
Registration wasn't fixed also in current version, because Add threw exception, didn't update it. I considered mods[ExternalMod.MakeKey(mod)] = mod, however it has side effect. If 2 dirs have the same mod id + version (key), it will be overwritten instead of thrown exception like so far.